### PR TITLE
style(oui-calendar): update timepicker style to specs

### PR DESCRIPTION
**Pull request by AxelPeter**
UK-66

* Update header style to specs
* Update timepicker style to specs

Related to https://github.com/ovh-ux/ovh-ui-angular/pull/302

### CODE REVIEW

File: packages/oui-calendar/_normalize.less [hunk 1]
```diff
@@ -44,6 +44,31 @@
...
+        font-weight: @oui-calendar-font-weight;
+        color: @oui-calendar-input-font-color;
+        background: @oui-calendar-input-background-color;
+
+        &:hover,
+        &:focus { background-color: @oui-calendar-input-background-color_hover; }
+      }
+
+      span {
+        background-color: @oui-calendar-input-background-color;
```
Comment (JeremyDec) [inline packages/oui-calendar/_normalize.less]:
```suggestion
   span {
         &,
         &:hover,
￼         &:focus { background-color: @oui-calendar-input-background-color; }
   }
```

File: packages/oui-calendar/_normalize.less [hunk 2]
```diff
@@ -249,5 +285,41 @@
...
+    .numInputWrapper {
+      width: ~"calc(@{oui-calendar-cell-width} * 2)";
+      height: 100%;
+      flex: 1;
+    }
+  }
+
+  &-time &-time-separator {
+
+  }
```
Comment (JeremyDec) [inline packages/oui-calendar/_normalize.less]: Empty selector ?

Comment (AxelPeter) [reply packages/oui-calendar/_normalize.less]: More an oversight :)

File: packages/oui-calendar/_normalize.less (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-calendar/_normalize.less]: Yes, could be. There was two different colors before, but it has been changed.

**Pull request merged**